blocks/algorithms/__init__.py

```diff
@@ -245,22 +248,18 @@ def compute_steps(self, gradients):
 
         Returns
         -------
-        An ordered dictionary of the same form as `gradient`, with the
-        proposed steps as values.
+        A tuple (steps, updates). `steps` is ordered dictionary of the
```

Comment:
Backquotes by themselves actually don't do much in Sphinx I think (they just make the text italic). If you want something to be monospace, you need to use double backquotes. In this case proper format should make Sphinx figure things out by itself, but this isn't proper format actually (we've been sloppy about that in large parts of the library). It should be:

```
Returns
-------
steps : OrderedDict
    Ordered dictionary of the same form as :attr:`gradients`
updates : list
    A list of tuples, representing the updates to perform.
```
or alternatively
```
Returns
-------
OrderedDict
    Ordered dictionary of the same form as :attr:`gradients`
list
    A list of tuples, representing the updates to perform.
```
If you want a link to `OrderedDict` to show up you could use ```:class:`~collections.OrderedDict```` in the first case. In the second version (with types only, no names) you can't use this because of a bug in Sphinx (https://bitbucket.org/birkenfeld/sphinx-contrib/issue/82/napoleon-return-type-containing-colons-is)